### DISCUSSION TITLE
Welcome to bitxhub Discussions!

### DISCUSSION THREAD

**Discussion by 0xforever9** [meshplus/bitxhub#280] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jzhe886** [comment]
good discussion💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.💪.!

**Comment by Aric112358** [comment]
So could the Bitxhub implement the cross-chain between btc and substrate? 
Coz as the figure illustrates, the Chain-to-chain architecture need both have the gatew
ay
